### PR TITLE
Remove Port 9250; Change 9300 to include CCS

### PR DESCRIPTION
**Pull request by HenryTheSir**
### Description
_Describe what this change achieves._
Changes Documentation for required Ports
### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._
Closes https://github.com/opensearch-project/documentation-website/issues/7100

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peterzhuamazon — CHANGES_REQUESTED** [review]
Need CCR team to confirm on this.
Their documentation: https://github.com/opensearch-project/cross-cluster-replication/?tab=readme-ov-file#step-1-start-test-clusters-with-replication-plugin-locally

cc: @saikaranam-amazon @ankitkala @monusingh-1 

Thanks.

**Review by monusingh-1 — APPROVED** [review]

**Comment by hdhalter** [comment]
Thanks, @HenryTheSir and @monusingh-1 !

File: _install-and-configure/install-opensearch/index.md [hunk 1]
```diff
@@ -60,8 +60,7 @@ Port number | OpenSearch component
 443 | OpenSearch Dashboards in AWS OpenSearch Service with encryption in transit (TLS)
 5601 | OpenSearch Dashboards
 9200 | OpenSearch REST API
-9250 | Cross-cluster search
-9300 | Node communication and transport
+9300 | Node communication and transport (Cluster internal), Cross Cluster Search
```
Comment (Naarcha-AWS) [inline _install-and-configure/install-opensearch/index.md:63]: What do we mean by "Cluster internal"? Is this port for the internal transport layer only?

```suggestion
9300 | Node communication and transport (internal cluster), Cross Cluster Search
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

File: _install-and-configure/install-opensearch/index.md (hunk 1 above)
Comment (hdhalter) [reply _install-and-configure/install-opensearch/index.md:63]: Should cross cluster search be lower case?

Comment (monusingh-1) [reply _install-and-configure/install-opensearch/index.md:63]: > What do we mean by "Cluster internal"? Is this port for the internal transport layer only?

Yes

Comment (ankitkala) [reply _install-and-configure/install-opensearch/index.md:63]: Change suggested by @Naarcha-AWS  looks good to me

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**